frontend/packages/ceph-storage-plugin/src/components/independent-mode/fileUpload.scss

```diff
@@ -0,0 +1,24 @@
+.upload-component {
+  display: inline-flex;
+  width: 100%;
+}
+
+.inputbtn {
```

Comment:
Minor - it should be `input-btn` as to make the format consistent across.

Reply:
ack